input/dispatch/dispatch.go

```diff
@@ -14,7 +14,7 @@ func Get(desire def.Input) input.Input {
 	case "dir":
 		input = dir.New(desire)
 	case "tar":
-		input = tar.New(desire)
+		input = tar2.New(desire)
```

Comment:
I think this is a mistake - or rather, incomplete. The package names are fine but we should keep a way to dispatch GNU-wrapping transports until they're **excessively** dead. I want to be able to trivially fall back on these for awhile.

I propose `gnu-tar` for the old one; thoughts?